tests/containers/helm.pm

```diff
@@ -0,0 +1,75 @@
+# SUSE's openQA tests
+#
+# Copyright 2022 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+#
+# Summary: Test deploy a helm chart
+#
+# Maintainer: qa-c team <qa-c@suse.de>
+
+use Mojo::Base 'publiccloud::basetest';
+use testapi;
+use utils qw(zypper_call script_retry);
+use version_utils qw(is_sle);
+use registration qw(add_suseconnect_product get_addon_fullname);
+
+sub run {
+    my ($self) = @_;
+
+    $self->select_serial_terminal;
+    $self->{deployment_name} = "apache";
+    my $chart = "bitnami/apache";
+
+    zypper_call("in -y jq aws-cli helm podman kubernetes-client");
+
+    # Check helm version
+    assert_script_run("helm version");
+    record_info('Helm version', script_output("helm version"));
+
+    # Connexion to AWS
+    my $provider = $self->provider_factory(service => 'EKS');
+    $self->{provider} = $provider;
+
+    # Access to Cluster
+    record_info('Kubectl version', script_output("kubectl version"));
+    assert_script_run("cat \$HOME/.kube/config");
+    assert_script_run("kubectl get nodes");
+
+    # Ensure is deleted
+    script_run("helm delete $self->{deployment_name}");
+
+    # Add repo, search and show values
+    assert_script_run("helm repo add bitnami https://charts.bitnami.com/bitnami");
+    assert_script_run("helm search repo apache");
+    assert_script_run("helm show all $chart");
+
+    # Install apache
+    assert_script_run("helm install $self->{deployment_name} $chart");
+    assert_script_run("helm list");
+
+    # Wait for deployment
+    assert_script_run("kubectl rollout status deployment/$self->{deployment_name}");
+    assert_script_run("kubectl describe deployment/$self->{deployment_name}");
+    assert_script_run("kubectl get pods | grep $self->{deployment_name}");
+    assert_script_run("kubectl describe services/$self->{deployment_name}");
+
+    # Test
+    enter_cmd("kubectl port-forward  services/$self->{deployment_name} 10001:80 &");
+    script_retry("curl http://localhost:10001", delay => 5, retry => 3);
+}
+
+sub post_fail_hook {
+    my ($self) = @_;
+    assert_script_run("helm delete $self->{deployment_name}");
```

Comment:
```suggestion
    script_run("helm delete $self->{deployment_name}");
```
Just to not fail the test here, in case the test fails before creating the heml chart.

Reply:
mmm I show that in a post_fail_hook, this line is executed inside a post_fail_hook https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/f02af1d6b443591159e95e0ca0ab330ec2f7b308/lib/publiccloud/eks.pm#L33 should we remove the assert_script_run from there too?

Reply:
Yes, probably `script_run` fits better in these cases... I'm not 100% sure how post_fail hooks treat failures.... 
Anyway `eks.pm` can be changed in a different PR.

Reply:
done